spec/controllers/search_controller_search_spec.rb

```diff
@@ -286,377 +321,3 @@
     end
   end
 end
-
-RSpec.describe SearchController, 'GET to #codes', type: :controller do
```

Comment:
These were extracted out into their own search file and actually duplicate other tests